resource-directory/service/deviceDirectory.go

```diff
@@ -188,7 +193,8 @@ func (dd *DeviceDirectory) GetDevices(req *pb.GetDevicesRequest, srv pb.GrpcGate
 	}
 
 	if len(devices) == 0 {
-		return status.Errorf(codes.NotFound, "not found")
+		log.Debug("DeviceDirectory.GetDevices.filterDevicesByUserFilters returns empty devices")
```

Comment:
Why is such a message logged?

Reply:
just to know which filter returns empty data